targets/TARGET_RENESAS/TARGET_RZ_A1XX/TARGET_RZ_A1H/device/os_tick_ostm.c

```diff
@@ -130,43 +130,43 @@ int32_t OS_Tick_Setup (uint32_t freq, IRQHandler_t handler)
   return (0);
 }
 
-/// Enable OS Tick.
-void  OS_Tick_Enable (void)
-{
+// Enable OS Tick.
+// void  OS_Tick_Enable (void)
```

Comment:
are these SDK files (thus only commented out) ? The ideal would be to have this removed, but if there is a reason to leave this dead code?